### PR TITLE
This PR fixes the issue reported in nvidia-cosmos/cosmos-transfer2.5 #17.

### PR DESCRIPTION
**Pull request by yuanfei-Wang**
This PR fixes the issue reported in nvidia-cosmos/cosmos-transfer2.5 #17
**Root Cause**
The problem occurs when the input video contains 221 frames while the control video contains only 144 frames. As a result, the final chunk of the control video is empty (0 frames).
When the padding mode is set to the default "reflect", the following code runs:
```
padding_frames = input_frames.flip(dims=[1])[:, :padding, :, :]
input_frames = torch.cat([input_frames, padding_frames], dim=1)
```
Because input_frames is empty, the padding operation never terminates, causing the process to hang.

**Fix**
An empty-frame check has been added before padding to detect this case early and raise a clear warning to users, advising them to verify that the input and control videos have matching frame counts.

### CODE REVIEW

**Comment by aryamancodes** [comment]
Thanks for your contribution! I'd like to test this internally before merging. I'll get back to this soon!
